camel-amq-fakeapp/README.adoc

```diff
@@ -35,8 +35,15 @@ Failure injection may be added with a future version.
 The container image can easily be build with the fabric8 build target, which uses the S2I strategy provided you are connected to OpenShift. Once you have cloned the git repository execute the following commands:
 
  $ cd auto-perf-test/camel-amq-fakeapp
+ Change amqp.host in the following file
```

Comment:
Good point. Can you move the instruction before the code part so that we only have the command there?